cmd/gateway/s3x/s3x_test.go

```diff
@@ -43,20 +46,36 @@ func (g *testGateway) Shutdown(ctx context.Context) error {
 
 var _ minio.ObjectLayer = &testGateway{}
 
-// getTestGateway returns a testGateway that implements minio.ObjectLayer.
+var pathOnce sync.Once
+var testPath string
+var testPathErr error
```

Comment:
might be cleaner to group these into one `var` block like:

```Go
var (
  _ minio.ObjectLayer = &testGateway{}
 pathOnce sync.Once
 testPath string
 testPathErr error
)
```

Reply:
sure